osx/qmk_toolbox/AppDelegate.m

```diff
@@ -232,11 +231,14 @@ - (void)loadMicrocontrollers {
 - (void)loadKeyboards {
     NSData * data = [NSData dataWithContentsOfURL:[NSURL URLWithString:@"http://compile.qmk.fm/v1/keyboards"]];
     NSError * error = nil;
-    NSArray * keyboards = [NSJSONSerialization JSONObjectWithData:data options:0 error:&error];
-    [_keyboardBox removeAllItems];
-    [_keyboardBox addItemsWithObjectValues:keyboards];
-    [_keyboardBox selectItemAtIndex:0];
-    _keyboardBox.enabled = YES;
+    if (data != nil) {
```

Comment:
How about we invert this and return if nil

Reply:
What for?

Reply:
Either way leads to the same outcome.  The question is which is better/cleaner?

Reply:
TBH, I'm fine either way, I've just been doing early returns lately.